src/modules/Security.rb

```diff
@@ -898,6 +905,10 @@ def sysctl_config
       @sysctl_config
     end
 
+    def sysctl_conflict?
+      @conflict ||= sysctl_config.conflict?
```

Comment:
That was the reason for caching it, and it seems that also the higher_precedence files are cached, thus, I guess it is safe enough, although we could avoid caching it which could be safer in case that the SysctlConfig.conflict? implementation change